### ISSUE TITLE
Simple CRUD?

### ISSUE THREAD

**Issue by rogueyoshi** [stavroskasidis/BlazorWithIdentity#21]
What's the best practice for adding simple CRUD to this? Is the dbContext reusable for storing data other than the auth related stuff, or should it be done entirely separately?

**Comment by stavroskasidis** [comment]
Yes, of course it is. Just add some DbSet properties to the dbcontext and then add a migration. It just already contains some default user tables for Identity.

**Comment by rogueyoshi** [comment]
> Yes, of course it is. Just add some DbSet properties to the dbcontext and then add a migration. It just already contains some default user tables for Identity.

Thank you! It looks like API scaffolding will even do some of that lifting.
